CHANGELOG.rst

```diff
@@ -1,6 +1,17 @@
 Changelog
 ---------
 
+0.1.5 (2024-04-15)
+******************
+
+- Launch training using the last build configuration from a project
```

Comment:
Training can now be launched using the most recent model configuration from a project.